presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -21,10 +21,433 @@
 
 namespace facebook::presto {
 
+#define SYS_CONF_BASE_PROPS(V)                                                \
```

Comment:
@yhwang What is the motivation to define this preprocessor? Why refactor this code? Thanks!